src/main/java/appeng/parts/p2p/ItemP2PTunnelPart.java

```diff
@@ -104,7 +70,7 @@ public ItemStack getStackInSlot(int slot) {
         @Override
         @Nonnull
         public ItemStack insertItem(int slot, @Nonnull ItemStack stack, boolean simulate) {
-            final ItemStack remainder = stack.copy();
+            int remainder = stack.getCount();
```

Comment:
Shouldn't we potentially use a fast return here for an empty stack? Technically we have one a couple lines down, but that is a bit hidden behind other conditions. Might also apply to anything with the concept of `empty`.

Reply:
We can, but I assume it would be done by the caller already.

Reply:
I don't think this is useful.